ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
This file needs to be called `optional-operators-configure-proxy-registry-commands.sh`